src/tcgwars/logic/impl/gen4/LegendsAwakened.groovy

```diff
@@ -4119,14 +4136,14 @@ public enum LegendsAwakened implements LogicCardInfo {
             text "Prevent all effects of attacks, including damage, done to Mewtwo by your opponent's Pokémon that isn't an Evolved Pokémon."
             delayedA {
               before null, self, Source.ATTACK, {
-                if (self.owner.opposite.pbg.active.evolution && bg.currentTurn==self.owner.opposite && ef.effectType != DAMAGE){
```

Comment:
Not necessarily specific to this card: How does this work with Lv.X basics? Also, basics with a card on top (not relevant to worlds formats, but there's Ditto BCR which could cause issues in unlimited/HoF)